src/components/navigation_bar.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+import { Nav, Navbar} from 'react-bootstrap';
+import logo from "../images/default.png"
```

Comment:
Instead of this PNG logo we could add the SVG logo which scales more efficiently as SVGs are [vector graphics](https://en.wikipedia.org/wiki/Vector_graphics). :smile: